### PR TITLE
SUP-12797-Player captions with layout set to below video gets cut off

### PR DESCRIPTION
**Pull request by ErezBouskila**

### CODE REVIEW

**Comment by ErezBouskila** [comment]
@OrenMe Please review PR

File: modules/KalturaSupport/resources/mw.ClosedCaptions.js [hunk 1]
```diff
@@ -218,7 +218,8 @@
 			});
 
 			if( this.getConfig('layout') == 'below'){
-				this.updateBelowVideoCaptionContainer();
+                this.getPlayer().triggerHelper("removeTextSize");
```
Comment (OrenMe) [inline modules/KalturaSupport/resources/mw.ClosedCaptions.js]: why bind this via event? you can know in advance you are in `below` mode and init the CVAA plugin accordingly: https://github.com/kaltura/mwEmbed/pull/3660/files#diff-322dfb3fd3c5e59a19ccacb340d0fc71L55

Comment (ErezBouskila) [reply modules/KalturaSupport/resources/mw.ClosedCaptions.js]: @OrenMe made the changes.

File: modules/KalturaSupport/resources/mw.ClosedCaptions.js [hunk 2]
```diff
@@ -217,11 +220,12 @@
 				}
 			});
 
-			if( this.getConfig('layout') == 'below'){
-				this.updateBelowVideoCaptionContainer();
+			if( this.getConfig('layout') === 'below'){
+                //this.getPlayer().triggerHelper("removeTextSize");
```
Comment (OrenMe) [inline modules/KalturaSupport/resources/mw.ClosedCaptions.js]: remove comment

Comment (ErezBouskila) [reply modules/KalturaSupport/resources/mw.ClosedCaptions.js]: Done

**Pull request merged**